pkg/gitlog/gitlog.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var (
-	lineRegex = regexp.MustCompile(`(?im)^\s*(closes?|related|relates|merges?|back\s?ports?|)\s+.*$`)
+	lineRegex = regexp.MustCompile(`(?im)^ *(closes?|related|relates|merges?|back\s?ports?|) +.*$`)
```

Comment:
🔧 I like the change, but I think we can improve it a bit further.

It also picks up issue references that are not marked with `closes` or `related` etc. See https://regex101.com/r/IOAQKD/1. 

I propose https://regex101.com/r/IOAQKD/3
```suggestion
	lineRegex = regexp.MustCompile(`(?im)^ *(closes?|related|relates|merges?|back\s?ports?) +.*$`)
```